controllers/registry.go

```diff
@@ -761,15 +761,19 @@ func (r *DPAReconciler) parseAzureSecret(secret corev1.Secret, secretKey string)
 	return azcreds, nil
 }
 
-func (r *DPAReconciler) getMatchedKeyValue(matchedKey string, line string) (string, error) {
-	cleanedLine := strings.ReplaceAll(line, " ", "")
-	cleanedLine = strings.ReplaceAll(cleanedLine, "\"", "")
-	matchedKeyValue := strings.Replace(cleanedLine, matchedKey, "", -1)
-	if len(matchedKeyValue) == 0 {
-		r.Log.Info("Could not parse secret for %s", matchedKey)
-		return matchedKeyValue, errors.New("azure secret parsing error")
+// Return value to the right of = sign
+func (r *DPAReconciler) getMatchedKeyValue(key string, s string) (string, error) {
+	s = strings.ReplaceAll(s, "\"", "")
```

Comment:
@kaovilai IMO we should remove the `"` `'`  as well from the string, WDYT ?

Reply:
Certainly.